src/curated-corpus/components/ImageUpload/ImageUpload.tsx

```diff
@@ -18,8 +18,8 @@ import { useStyles } from './ImageUpload.styles';
 import { useNotifications } from '../../../_shared/hooks';
 import {
   ApprovedCorpusItem,
-  MutationUploadApprovedCorpusItemImageArgs,
```

Comment:
After effects of merging `main` in to this branch? I think there should be no changes to this file in this PR

Reply:
Yes I resolved the conflicts in the browser so the formatting is a bit off but I can revert this. Seems like I moved that line to one below